Source/WTF/wtf/MallocHeapBreakdown.cmake

```
@@ -11,3 +11,14 @@ target_include_directories(MallocHeapBreakdown
     PUBLIC
     malloc_heap_breakdown
 )
+
+if (USE_SYSPROF_CAPTURE)
+  target_link_libraries(MallocHeapBreakdown
+    PRIVATE
+    SysProfCapture::SysProfCapture
+  )
+  target_compile_definitions(MallocHeapBreakdown
+    PUBLIC
```

I think this can be `PRIVATE`, because the `USE_SYSPROF_CAPTURE` macro is only checked from the `main.cpp` file is not needed anywhere else.


```suggestion
    PRIVATE
```